lib/neo4j/active_node/query/query_proxy.rb

```diff
@@ -1,7 +1,9 @@
 module Neo4j
   module ActiveNode
     module Query
+      # rubocop:disable Metrics/ClassLength
```

Comment:
`rubocop:disable` makes @cheerfulstoic :sadpanda:

Not saying this can't be fixed later ;p